app/mailers/instructeur_mailer.rb

```diff
@@ -47,4 +47,21 @@ def send_notifications(instructeur, data)
   def self.critical_email?(action_name)
     action_name == "send_login_token"
   end
+
+  def confirm_and_notify_added_instructeur(instructeur, group, current_instructeur_email)
+    @instructeur = instructeur
+    @group = group
+    @current_instructeur_email = current_instructeur_email
+    @reset_password_token = instructeur.user.send(:set_reset_password_token)
+
+    subject = if group.procedure.groupe_instructeurs.many?
+      "Vous avez été ajouté(e) au groupe \"#{group.label}\" de la démarche \"#{group.procedure.libelle}\""
```

Comment:
Joli la distinction !

```suggestion
      "Vous avez été ajouté(e) au groupe « #{group.label} » de la démarche « #{group.procedure.libelle} »"
```